source/linear-algebra/source/03-AT/06.ptx

```diff
@@ -117,17 +117,17 @@ was actually defined for all kinds of vector spaces:
 Let <m>V</m> be a vector space with the basis <m>\{\vec v_1,\vec v_2,\vec v_3\}</m>.
 Which of these completes the following definition for a bijective linear map
 <m>T:V\to\mathbb R^3</m>?
-<me>T(\vec v)=T(a\vec v_1+b\vec v_2+c\vec v_3)=\left[\begin{array}{c}
+<me>T(\vec v)=T(a\vec v_1+b\vec v_2+c\vec v_3)=\unknown\vec e_1+\unknown\vec e_2+\unknown\vec e_3=\left[\begin{array}{c}
     \unknown\\\unknown\\\unknown
   \end{array}\right]</me>
   <ol marker="A." cols="3">
```

Comment:
```suggestion
  <ol marker="A.">
```

Reply:
on the preview links, the answers are all squished. If I'm not mistaken, this suggestion should stack them one on top of another?